cmd/system-probe/config/config.go

```diff
@@ -138,7 +139,7 @@ func load() (*Config, error) {
 
 	// this check must come first, so we can accurately tell if system_probe was explicitly enabled
 	npmEnabled := cfg.GetBool("network_config.enabled")
-	usmEnabled := cfg.GetBool(key(smNS, "enabled"))
+	usmEnabled := cfg.GetBool(key(smNS, "enabled")) || cfg.GetBool(key(dsmNamespace, "enabled"))
```

Comment:
Why is this getting its own top-level config namespace? Is it a separate product from USM?

Reply:
If it is a separate product, please create a new variable `dsmEnabled` and update the logic below. Otherwise it is quite confusing how these config options control the behavior.

Reply:
Yes, create a `dsmEnabled` variable, and then use it below to turn on the modules necessary. This is similar to how USM works, since USM requires NPM, but doesn't enable billing for it.